designsafe/apps/data/models/taxonomy.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+from django.db import models
+from django.contrib.auth.models import User
+from django.contrib.auth.validators import ASCIIUsernameValidator
+
+
+
+
+class CustomUser(User):
+    username_validator = ASCIIUsernameValidator()
+
+class Vocab(models.Model):
+    Term = models.CharField(max_length=40, unique=True)
```

Comment:
Class attributes have to be PEP8 compliant. i.e. `term`, `definition`, `source_of_definition`, `date` and `vocab_builder`